sys/console/full/src/console.c

```diff
@@ -473,10 +474,14 @@ console_handle_char(uint8_t byte)
         case ESC:
             esc_state |= ESC_ESC;
             break;
-        default:
-            insert_char(&input->line[cur], byte, end);
-            /* Falls through. */
         case '\r':
```

Comment:
/* Falls through. */ comment is needed here as GCC7 gives an error without an explicit commented fallthrough.